### DISCUSSION TITLE
Voice notes

### DISCUSSION THREAD

**Discussion by DeeKahy** [TechbeeAT/jtxBoard#291] [Ideas]
I would like to see the option to make and save voice notes to listen to later.

**Comment by patrickunterwegs** [comment]
Hey @DeeKahy , you mean like an option as it already exists for linked notes in the detail view of an entry?

**Reply by DeeKahy** [reply]
Yes, I wasn't aware that the audio option was already available for linked notes. However, I still think it would be great to have it as a more prominent option.

**Reply by patrickunterwegs** [reply]
Alright, thanks! Indeed I get it that it makes sense to have the option for audio notes also on the top level. 

In fact I implemented it for subtasks also to see if it would basically be used, so now I have the confirmation 😉

**Reply by patrickunterwegs** [reply]
Hey @DeeKahy !
I've been thinking a bit about this, my approach is to add another icon at the bottom of the page next to the lightning and pen symbol.

The button behavior could be:
1. The same dialog opens like the one that already exists for subnotes.
2. You hold the button and the recording starts directly and creates a new entry when you release it.

What is your opinion? What would be better?
I tend a bit to 1 as it gives a bit more control to the user and also you can discard a faulty recording more easily.
On the other hand 2 would be an approach that allows users to make audio notes really quickly...

**Reply by DeeKahy** [reply]
I think option 2 could be improved by adding a slide right or left gesture whilst recording that discards the current note, since I catch myself making mistakes whilst still recording (kinda like telegram does it).

But as the options currently stand (without the slide thing) I think option 1 is slightly better and would be easier to implement.

**Reply by DeeKahy** [reply]
I wish I could edit replies...
Sorry for the 2nd message but I forgot to say that I think having another icon at the bottom is a great idea.

**Reply by patrickunterwegs** [reply]
Hey @DeeKahy ! 
I have implemented audio notes now also for main entries (not only audio notes for linked notes). It will be available in v2.4.0 🙂
